linkml/generators/rustgen/rustgen.py

```diff
@@ -565,6 +601,47 @@ def render(self, mode: Optional[RUST_MODES] = None) -> Union[FileResult, CrateRe
             res = FileResult(file=file, source=sv.schema)
             return res
 
+    def gen_poly_trait(self, cls: ClassDefinition) -> PolyTrait:
```

Comment:
some documentation would be good for this. what is PolyTrait? why is it needed? what does it do?